packages/time-field/stories/time-field.stories.ts

```diff
@@ -0,0 +1,377 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { html, nothing, TemplateResult } from '@spectrum-web-components/base';
+import { TimeGranularity } from '@spectrum-web-components/input-segments/src/types.js';
+
+import { spreadProps } from '../../../test/lit-helpers.js';
+
+import '@spectrum-web-components/help-text/sp-help-text.js';
+import '@spectrum-web-components/time-field/sp-time-field.js';
+import '@spectrum-web-components/theme/sp-theme.js';
+
+const locales = [
```

Comment:
In that some of these has drastic visual delivery differences, we will need to work out the right ones to include (and how) in the VRT process to ensure delivery long term...

Reply:
What I can say is that the field content changes significantly when we have some locales defined. The field itself doesn't change, the "wrapper", but the content definitely does.